src/index.ts

```diff
@@ -282,6 +282,11 @@ export class Bundler {
 
     const env = Object.assign({}, config.env)
 
+    // drop precess.env.NODE_ENV from umd/iife
+    if (['umd', 'umd-min', 'iife', 'iife-min'].includes(format)) {
+      env.NODE_ENV = minify ? 'production' : 'development'
+    }
```

Comment:
we can skip this if `env.NODE_ENV` is already set,

Reply:
done